openapi_tester/exceptions.py

```diff
@@ -8,26 +8,30 @@ class DocumentationError(AssertionError):
     Custom exception raised when package tests fail.
     """
 
-    def __init__(
+    def __init__(  # pylint: disable=too-many-arguments
         self,
         message: str,
         response: Any,
         schema: dict,
         hint: str = "",
         reference: str = "",
+        show_expected=True,
     ) -> None:
         from openapi_tester.schema_converter import SchemaToPythonConverter
 
-        converted_schema = SchemaToPythonConverter(schema, with_faker=False).result
-        super().__init__(
-            self.format(
-                response=self._sort_data(response),
-                example_item=self._sort_data(converted_schema),
-                hint=hint,
-                message=message,
-                reference=reference,
+        if show_expected:
```

Comment:
this is not a good solution IMO. 

Two options:

1. create a root DRFTesterException and then sub-class it. 
2. make the parameters optional in this exception, and based on (a) if we have a schema passed, a response passed etc, run format or do not.
3. I personally think that option 1 is better - more pythonic/standard on the one hand, and less complex. The errors in this library are by far the most verbose I have seen :) and the format method here is overengineered IMHO. I.E. i'd strip this to the essential.

Reply:
Yeah I don't disagree with this. A base exception sounds like a good solution; I'll give it another try tonight.

Reply:
well, please fix this in the scope of this PR - not a separate one. I do not want this "show_expected" stuff to be merged at all.

Reply:
Ok no problem.